app/services/oregon_digital/verify_work_service.rb

```diff
@@ -43,7 +43,7 @@ def resource_type
     end
 
     def work
-      @work ||= resource_type.constantize.find(@args[:pid])
+      @work ||= Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: @args[:pid])
```

Comment:
A note for querying valkyrie:
We still have some issues (like indexing) that require an AF object. If this starts breaking, you can add `use_valkyrie: false` as an arg to return an AF object, but still keep us prepared to switch over once we have real valkyrie objects in place